### PR TITLE
Qubes 4.0 Getting Started

### PR DESCRIPTION
**Pull request by marmarta**
A new Getting Started page for Qubes 4.0 - with pictures (requires  https://github.com/QubesOS/qubes-attachment/pull/16 ) and updated text.

### CODE REVIEW

**Comment by andrewdavidwong** [comment]
Thank you for doing this, @marmarta. Looks good overall. I'll push some revisions when I get a chance.

One problem: According to our [doc guidelines](https://www.qubes-os.org/doc/doc-guidelines/), this should technically be a section on the existing Getting Started page (alongside the 3.2 version). However, this would be rather awkward, and the Getting Started page might be a legitimate exception to this rule for independent reasons. On the other hand, having two separate pages is also awkward, because the existing Getting Started page is linked from many places, and many of those places don't easily allow for two separate links to two different versions. We shouldn't remove the current Getting Started page until 3.2 is no longer supported. Do you or @marmarek have any thoughts about how to address this?

Since 3.2 is approaching EOL soon anyway, one idea is to put this new page in place of the existing one and provide a link at the top pointing to the old one, e.g.:

*This is an introduction to Qubes 4.0. Looking to get started with Qubes 3.2? Please click **here**.*

**Comment by marmarek** [comment]
> Since 3.2 is approaching EOL soon anyway, one idea is to put this new page in place of the existing one and provide a link at the top pointing to the old one, e.g.:
> 
> _This is an introduction to Qubes 4.0. Looking to get started with Qubes 3.2? Please click **here**._

Good idea.

**Comment by andrewdavidwong** [comment]
GitHub won't let me request a review from @marmarta, only from @marmarek. However, I'd like you both to take a look at the changes I've made to make sure you're comfortable with them.

**Comment by marmarta** [comment]
Looks good from here!

**Comment by andrewdavidwong** [comment]
After installing 4.0.1, I noticed that we seem to use "qube" to refer to any kind of VM (including TemplateVMs?), not just AppVMs. If that's the case, then "qubes and templates" doesn't make sense.

- If we use "AppVM," then we might as well use "TemplateVM" and "VM" in general.
- We can't use "apps and templates" because "app" is ambiguous between AppVMs and applications.
- If we use "app qube," then we should also uses "template qube" for consistency.

**Comment by marmarta** [comment]
True enough; I prefer to use 'qube' (because it seems VM scares away some folk), but for others 'VM' is much clearer. Which one would you prefer?

**Comment by andrewdavidwong** [comment]
> True enough; I prefer to use 'qube' (because it seems VM scares away some folk), but for others 'VM' is much clearer. Which one would you prefer?

Well, I personally prefer "VM," but we've been discussing it for years now (see #1015). The problem is that if "qube" is supposed to mean some _proper subset_ of VMs, then it's hard to guard against lax misuse of the term as a synonym for "VM," which leads to inconsistency.

So, suppose it's just a synonym for "VM." Then we can't say there are two main kinds of things -- qubes and templates -- because templates are a _type_ of qube. So what do we call the first type of thing (previously known as "AppVM")?

**Comment by unman** [comment]
On Mon, Jan 14, 2019 at 08:19:48PM -0800, Andrew David Wong wrote:
> > True enough; I prefer to use 'qube' (because it seems VM scares away some folk), but for others 'VM' is much clearer. Which one would you prefer?
> 
> Well, I personally prefer "VM," but we've been discussing it for years now (see #1015). The problem is that if "qube" is supposed to mean some _proper subset_ of VMs, then it's hard to guard against lax misuse of the term as a synonym for "VM," which leads to inconsistency.
> 
> So, suppose it's just a synonym for "VM." Then we can't say there are two main kinds of things -- qubes and templates -- because templates are a _type_ of qube. So what do we call the first type of thing (previously known as "AppVM")?

qube is supposed to be a friendlier term for non technical users. For
that audience, I think that the qube/template distinction works
reasonably well. I don't think that qube is intended to be a synonym for
"VM", and we should use it throughout.
In technical documentation, and in discussions where appropriate, then
VM maybe used, with clarifications as necessary to distinguish
templates/appVMs and qubes not based on a template.

If there were some example where someone could be shown to be confused by
the use of "qube" this would be helpful. I am not aware of any such
case, but might have missed it.

**Comment by andrewdavidwong** [comment]
> qube is supposed to be a friendlier term for non technical users. For
that audience, I think that the qube/template distinction works
reasonably well. I don't think that qube is intended to be a synonym for
"VM", and we should use it throughout.
> [...]
> If there were some example where someone could be shown to be confused by
the use of "qube" this would be helpful. I am not aware of any such
case, but might have missed it.

The Qube Manager manages all VMs, including TemplateVMs. Most of the actions it advertises (start/stop qube, delete qube, clone qube, backup qubes) also apply to templates. So, it's easy for users to get the idea that "qube" can refer to any type of VM, including templates.

There are many ways a term that is ambiguous between "AppVM" and "VM" can be confusing. Here are some examples of statements that are true on one definition but false on the other:
 - "In general, software installed in a qube will not persist across restarts."
 - "There's no need to update your qubes."
 - "A qube is intended for activities like web browsing and video chats."
 - "There's a special kind of qube called a 'template' where you install your apps."
 - "Every qube is based on a template."
 - "Not every qube is based on a template."

File: basics_user/getting-started.md [hunk 1]
```diff
@@ -9,200 +9,200 @@ redirect_from:
...
-By default all qubes are based on a single, common **TemplateVM** , although you can create more TemplateVMs if you wish. 
+Introduction
+------------
+
+In Qubes OS, you run all your programs in lightweight [virtual machines (VMs)] called [qubes].
+Not every app runs in its own qube.
+(That would be a big waste of resources!)
+Instead, each qube represents a [security domain] (e.g., "work," "personal," and "banking").
+By default, all qubes are based on a single, common [template], although you can create more templates if you wish. 
 When you create a new qube, you don't copy the whole root filesystem needed for this qube to work (which would include copying all the programs).
```
[the remark this answers is not in the document]
Comment (andrewdavidwong) [reply basics_user/getting-started.md]: Done.

File: basics_user/getting-started.md [hunk 2]
```diff
@@ -9,200 +9,200 @@ redirect_from:
...
+Instead, each qube *shares* the root filesystem with its respective template. 
+A qube has read-only access to the filesystem of the template on which it's based, so a qube cannot modify a template in any way. 
+This is important, as it means that if a qube is ever compromised, the template on which it's based (and any other qubes based on that template) will still be safe. 
+So, creating a large number of qubes is cheap: each one needs only as much disk space as is necessary to store its private files (e.g., the "home" folder).
+
+If you've installed Qubes OS using the default options, a few qubes have already been created for you:
+
+ - work
+ - personal
+ - untrusted
```
[the remark this answers is not in the document]
Comment (andrewdavidwong) [reply basics_user/getting-started.md]: Done.

**Comment by marmarta** [comment]
why has this not been merged in the end? does it need another rewrite/edit?

**Comment by andrewdavidwong** [comment]
> why has this not been merged in the end? does it need another rewrite/edit?

It looks like it's waiting on review approval from @marmarek.

**Comment by andrewdavidwong** [comment]
I decided to just go ahead and merge this. Sorry it took so long, @marmarta. Next time will be faster.

**Pull request merged**